pytensor/tensor/math.py

```diff
@@ -681,39 +681,32 @@ def largest(*args):
         return max(stack(args), axis=0)
 
 
-@scalar_elemwise
 def lt(a, b):
-    """a < b"""
+    """Elementwise less-than comparison, `a < b`."""
 
 
-@scalar_elemwise
 def gt(a, b):
-    """a > b"""
+    """Elementwise greater-than comparison, `a > b`."""
 
 
-@scalar_elemwise
 def le(a, b):
-    """a <= b"""
+    """Elementwise less-than-or-equal-to comparison, `a <= b`."""
 
 
-@scalar_elemwise
 def ge(a, b):
-    """a >= b"""
+    """Elementwise greater-than-or-equal-to comparison, `a >= b`."""
 
 
-@scalar_elemwise
 def eq(a, b):
-    """a == b"""
+    """Elementwise equality comparison, `a == b`."""
```

Comment:
Adding a special note for `eq` and `neq` which can't be overloaded (while >, >= and alike are overloaded just fine)

```suggestion
    """Elementwise equality comparison, `a == b`.
    
    Due to Python rules it is not possible to correcly overload the equality symbol `==`,
    so `eq` must always be used to compute the Elemwise equality of TensorVariables.
    """
```